spec/factories/entities.rb

```diff
@@ -0,0 +1,11 @@
+require 'faker'
+
+FactoryBot.define do
+  factory :entity do
+    name { Faker::Company.name }
+    amount { Faker::Number.decimal(l_digits: 2) }
```

Comment:
- [ ] Please include integration tests currently you have none. Also be reminded: 
![image](https://github.com/ivanmvh/rails_wallet/assets/62705947/bf7e5b08-9038-4e2b-a3bc-d9bd47a19bf7)